app/views/assays/_show.html.erb

```diff
@@ -0,0 +1,121 @@
+<% single_page ||= false %>
```

Comment:
I think this is not a good way of doing things, it involves adding this option to too many partials.

Instead, you could set a variable in the controller, and use the `helper_method` function to make it globally available to views, like what is done for `controller_model`:
https://github.com/seek4science/seek/blob/6428fe4908958b1fe1e2a142010bf2e2cf3d820c/app/controllers/application_controller.rb#L128-L135